ui/components/Expenses/ExpenseItem.tsx

```diff
@@ -18,9 +21,15 @@ export default function ExpenseItem(props: Props): JSX.Element {
   } = useStore();
 
   const isSelected = expense.spendingId === selectedExpenseId;
-  const fundingSchedule = useFundingSchedule(expense.fundingScheduleId);
+  const spendingFunding = useSpendingFunding(expense);
+  const fundingSchedules = useFundingSchedules();
+
   const onClick = () => setCurrentExpense(isSelected ? null : expense.spendingId);
 
+  const nextFunding = spendingFunding.result.length > 0 && spendingFunding.result
+    .sort((a, b) => fundingSchedules.get(a.fundingScheduleId).nextOccurrence.unix() > fundingSchedules.get(b.fundingScheduleId).nextOccurrence.unix() ? 1 : -1)[0]
+  const fundingSchedule = nextFunding && fundingSchedules.get(nextFunding.fundingScheduleId);
```

Comment:
Leave a comment about what's going on here.

What happens if we get two funding schedules that land on the same day?